packages/firebase_messaging/firebase_messaging_web/lib/src/interop/messaging.dart

```diff
@@ -8,6 +8,7 @@
 import 'dart:async';
 import 'dart:js_interop';
 
+import 'package:_flutterfire_internals/_flutterfire_internals.dart';
```

Comment:
Why do we need this extra import?

Reply:
Opps I was trying something, and forgot to delete it.